app/code/Magento/Checkout/Block/Cart/Shipping.php

```diff
@@ -88,13 +88,4 @@ public function getBaseUrl()
     {
         return $this->_storeManager->getStore()->getBaseUrl();
     }
-
```

Comment:
Hello @jonathanKingston. We can't remove public and protected methods from classes that marked as API by backward compatibility reason. We should mark those methods as deprecated.
You can read more about our backward compatibility here 
http://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/index.html